src/identity/apis/org.ts

```diff
@@ -117,6 +118,19 @@ export const fetchDefaultAccountDefaultOrg = async (): Promise<
   }
 }
 
+// fetch data regarding whether the user can create new orgs, and associated upgrade options.
+export const fetchOrgCreationAllowance = async () => {
+  const response = await getAllowancesOrgsCreate({})
+
+  if (response.status !== 200) {
+    throw new GenericError(
+      'Failed to determine whether this user can create a new organization.'
+    )
+  }
```

Comment:
I'll check with Randy on whether there should be any other error codes when the endpoint is created. Right now it's just a `200`, but I would assume we should ultimately be able to get at least a `401` if the user isn't logged in, or a `500` if there is some unforeseen server issue.